openapi.yaml

```diff
@@ -0,0 +1,88 @@
+openapi: 3.1.0
+info:
+  version: 0.1.0
+  title: AsyncAPI Server API
+  description: Server API providing official AsyncAPI tools
+  contact:
+    name: AsyncAPI Initiative
+    email: info@asyncapi.io
+    url: https://asyncapi.com/
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
+
+servers:
+  - url: http://api.asyncapi.io
```

Comment:
We should change it, but for what? (:

Reply:
Hah, thanks! But I had in mind for what domain 😅

Reply:
> Hah, thanks! But I had in mind for what domain 😅

Yeah, and for me `api.asyncapi.io` looks ok.

Reply:
Use `api.asyncapi.com` instead. We should start getting rid of the `.io` domain. It was used for historical reasons.